### DISCUSSION TITLE
Add Bibliography / reference list from code

### DISCUSSION THREAD

**Discussion by Placeprom** [dolanmiu/docx#1759] [Q&A] [answered]
Hey there!
I haven´t found a way to add a bibliography / reference list from code in the api docs / usage guidelines. Is there a way to add a bibliography and connect paragraphs from code or doesnt the package support this?

**Comment by dolanmiu** [comment, marked as answer]
There is a feature request for it already
https://github.com/dolanmiu/docx/issues/204

**Reply by dolanmiu** [reply]
Not available right now though

**Reply by Placeprom** [reply]
Thanks for getting back to me! I will have a look at the feature request in the next couple of days. I am also not sure if I can provide a pull request for this but I would like to try it :)
